src/components/SearchableSelect/SearchableSelect.tsx

```diff
@@ -438,8 +507,9 @@ const SearchableSelect = createClass({
 					)}
 					isDisabled={isDisabled}
 					onSelect={onSelect}
-					selectedIndices={isItemSelected ? [selectedIndex] : []}
-
+					selectedIndices={
+						isItemSelected && selectedIndex ? [selectedIndex] : []
```

Comment:
If `selectedIndex = 0`, this condition will be false. Currently this breaks in the docs if you select the first item in the list. Might want to just use `_.isNumber` since typescript can't pick up that `isItemSelected` implies selectedIndex is not null.

Reply:
good find. got that fixed up and tested in the docs